src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -7165,6 +7165,79 @@ components:
             $ref:
               '#/components/schemas/SnapshotBuilderDatasetConceptSet'
 
+    SnapshotBuilderOption:
```

Comment:
I see how changing these definitions changes the definition of snapshotBuilderSettings. But, I am still wondering about the process of populating the settings.

Reply:
Let me see if I understand correctly: For now, we're just going to manually populate them when we setup the snapshot builder settings. They'll be included in the JSON. Eventually, we want to regenerate them in the code.